utils/build/docker/ruby/rack/config.ru

```diff
@@ -7,7 +7,15 @@ require 'json'
 
 # tracer configuration of Rack integration
 
-require 'datadog/auto_instrument'
+begin
+  require 'ddtrace/auto_instrument'
+rescue LoadError
+end
+
+begin
+  require 'datadog/auto_instrument'
+rescue LoadError
+end
```

Comment:
I think we should unify those:

```ruby
begin
  require 'datadog/auto_instrument'
rescue LoadError
  require 'ddtrace/auto_instrument'
end
```

This was we know that at least one of them was loaded (or another `LoadError` will be raised)

Reply:
Thanks, I'll do that in Sinatra and parametric weblogs too then

Reply:
Fixed in 801bd8ab996e8465a2fe20ddcebaebc55a15595c